extensions/2.0/Khronos/KHR_techniques_webgl/schema/technique.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "Technique",
+    "type" : "object",
+    "description" : "A template for material appearances.",
+    "allOf" : [ { "$ref" : "glTFChildOfRootProperty.schema.json" } ],
+    "properties": {
+        "program" : {
+            "allOf" : [ { "$ref" : "glTFid.schema.json" } ],
+            "description" : "The index of the program."
+        },
+        "attributes" : {
+            "type" : "object",
+            "description" : "A dictionary object of `Attribute` objects.",
+            "properties" : {
+            },
```

Comment:
Throughout, for empty property objects just condense to a single line: `"properties" : {},`